packages/circuits/circom/coordinator/qv/tallyVotes.circom

```diff
@@ -275,3 +277,5 @@ template ResultCommitmentVerifier(voteOptionTreeDepth) {
     
     computedNewTallyCommitment === newTallyCommitment;
 }
+
+
```

Comment:
This additional new line can be removed. The issue is related to newly added files, not modified.